include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -171,7 +171,9 @@ NOTE(record_not_automatically_importable, none, "record '%0' is not "
                                                 "Manual to classify this type.",
      (StringRef, StringRef))
 NOTE(projection_not_imported, none, "C++ method '%0' that returns unsafe "
-                                    "projection of type '%1' not imported",
+                                    "projection of type '%1' not imported. "
+                                    "Do you want to replace it with a call "
+                                    "to the subscript operator?",
      (StringRef, StringRef))
```

Comment:
We usually use `DescriptiveDeclKind, Identifier` in cases like this instead of spelling out `method` and quotes.

Reply:
I think it's important that we explicitly say that this is a C++ method, so I'm not sure `DescriptiveDeclKind` will work.